bitbucket-pipelines.yml

```diff
@@ -8,7 +8,7 @@ pipelines:
           - apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys D2486D2DD83DB69272AFE98867170598AF249743
           - apt-get update
           - apt-get -y install
-            cmake build-essential lcov curl mercurial libprotobuf-dev protobuf-compiler libprotoc-dev
+            cmake build-essential lcov curl git libprotobuf-dev protobuf-compiler libprotoc-dev
```

Comment:
Should we be deleting this file?

Reply:
Jenkins CI still needs it for now just to get the Ubuntu version:

https://github.com/ignition-tooling/release-tools/blob/62a6ed795f4c2fad04c8ee444a8615512286a1a9/jenkins-scripts/dsl/ignition.dsl#L241-L248